forc/src/cli/commands/test.rs

```diff
@@ -70,7 +70,6 @@ pub(crate) fn exec(command: Command) -> Result<()> {
 
     // Pass through cargo test args.
     if !command.cargo_test_args.is_empty() {
-        cmd.arg("--");
```

Comment:
Could you elaborate on why this was removed?

The `--` separator is necessary in order to distinguish between the cargo test options and cargo test arguments. Perhaps you removed it when you were trying to pass some cargo test options to `--cargo-test-args`, when you perhaps instead wanted to use `--cargo-test-opts`?

Reply:
Jup! That is right. When I wanted to use `` --features``, I ran into problems until I deleted `` --``.